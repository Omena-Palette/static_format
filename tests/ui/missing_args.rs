use static_format::const_format;

fn main() {
    let _out = const_format!("hello world missing arguments {} and {}", 5);
}
