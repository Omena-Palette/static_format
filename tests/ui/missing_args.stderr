error: Wrong number of arguments, expected: 2 but found 1
 --> tests/ui/missing_args.rs:4:16
  |
4 |     let _out = const_format!("hello world missing arguments {} and {}", 5);
  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `const_format` (in Nightly builds, run with -Z macro-backtrace for more info)
